lib/Parser.js

```diff
@@ -1817,17 +1802,9 @@ class Parser extends Tapable {
 	}
 
 	evaluate(source) {
-		const ast = acorn.parse("(" + source + ")", {
-			ranges: true,
-			locations: true,
-			ecmaVersion: ECMA_VERSION,
-			sourceType: "module",
-			plugins: {
-				dynamicImport: true
-			}
+		const ast = Parser.parse("(" + source + ")", {
```

Comment:
I believe for evaluating we don't need ranges and locations. So you can pass `ranges: false, locations: false` as additional options.